### PR TITLE
Link to extracted files from tools page

### PR DESCRIPTION
**Pull request by dthaler**
Addresses part of https://github.com/FamilySearch/GEDCOM/issues/418

### CODE REVIEW

**Review by tychonievich — APPROVED** [review]

File: _pages/tools.md [hunk 1]
```diff
@@ -66,6 +66,23 @@ The full set of YAML files, together with some synthesis information such as par
...
+| [grammar.gedstruct](https://github.com/FamilySearch/GEDCOM/blob/main/extracted-files/grammar.gedstruct) | This file contains all of the blocks from the specification in the structure organization metasyntax concatenated into one file. |
+| [payloads.tsv](https://github.com/FamilySearch/GEDCOM/blob/main/extracted-files/payloads.tsv) | This tab-separated-values (TSV) file contains (structure type, payload type) pairs. |
+| [substructures.tsv](https://github.com/FamilySearch/GEDCOM/blob/main/extracted-files/substructures.tsv) | This tab-separated-values (TSV) file contains (superstructure type, substructure tag, substructure type) triples. |
+| [tags](https://github.com/FamilySearch/GEDCOM/blob/main/extracted-files/tags) | This directory contains YAML files for all URI-identified terms defined in the specifiction, including structure types, enumeration values and sets, calendar identifiers, and month names. There is a proposal to combine this directory into the [Extension Registry](#extension-registry), but that move has not fully taken place. |
```
Comment (dthaler) [inline _pages/tools.md:84]: Need to discuss this line

Comment (dthaler) [inline _pages/tools.md:84]:
```suggestion
| [tags](https://github.com/FamilySearch/GEDCOM/blob/main/extracted-files/tags) | This directory contains YAML files for all URI-identified terms defined in the specification, including structure types, enumeration values and sets, calendar identifiers, and month names. There is a proposal to combine this directory into the [Extension Registry](#extension-registry), but that move has not fully taken place. |
```

File: _pages/tools.md [hunk 2]
```diff
@@ -66,6 +66,23 @@ The full set of YAML files, together with some synthesis information such as par
...
+| [grammar.gedstruct](https://github.com/FamilySearch/GEDCOM/blob/main/extracted-files/grammar.gedstruct) | This file contains all of the blocks from the specification in the structure organization metasyntax concatenated into one file. |
+| [payloads.tsv](https://github.com/FamilySearch/GEDCOM/blob/main/extracted-files/payloads.tsv) | This tab-separated-values (TSV) file contains (structure type, payload type) pairs. |
+| [substructures.tsv](https://github.com/FamilySearch/GEDCOM/blob/main/extracted-files/substructures.tsv) | This tab-separated-values (TSV) file contains (superstructure type, substructure tag, substructure type) triples. |
+| [tags](https://github.com/FamilySearch/GEDCOM/blob/main/extracted-files/tags) | This directory contains YAML files for all URI-identified terms defined in the specification, including structure types, enumeration values and sets, calendar identifiers, and month names. There is a proposal to combine this directory into the [Extension Registry](#extension-registry), but that move has not fully taken place. |
```
Comment (dthaler) [inline _pages/tools.md:84]:
```suggestion
| [tags](https://github.com/FamilySearch/GEDCOM/blob/main/extracted-files/tags) | This directory contains YAML files for all URI-identified terms defined in the specification, including structure types, enumeration values and sets, calendar identifiers, and month names. The contents of this directory is mirrored into directories in the [Extension Registry](#extension-registry).
```

**Comment by dthaler** [comment]
Agreed during meeting 1 FEB 2024

**Pull request merged**